introduction/setting-up-eslint/README.md

```diff
@@ -0,0 +1,27 @@
+In this lesson, we will learn to setup ESLint extension for Visual Studio Code.
+
+Visual Studio Code provides a lot of inbuilt features to improve the speed at which we write code. Additionally, we can install extensions for Visual Studio Code that can help with code quality and readability.
+
+The ESLint Visual Studio Code extension, which we will use in this lesson, will also help with other JavaScript related projects.
+
+## Code Quality using ESLint
+
+Code quality is an important aspect, whenever we write code in any programming language. Writing quality code helps you limit issues and errors in your application.
+
+Identifying and fixing issues, that you might have missed while coding, can be streamlined with the help of Code Quality extensions like `ESLint`.
+
+ESLint analyses your code to quickly find problems. It is a common tool for identifying and reporting on coding patterns found in JavaScript code. The goal of ESLint is to make code more consistent while avoiding bugs. Many problems or issues highlighted by ESLint, can be automatically fixed.
```

Comment:
It is ~~a common~~ the standard tool for identifying and reporting ~~on~~ coding patterns

~~Many problems or issues highlighted by ESLint, can be automatically fixed.~~ ESLint is intelligent enough to suggest fixes to the most problems it finds out.